index.html

```diff
@@ -270,87 +329,289 @@
 It will continue to evolve and the task force will issue updates as often as needed. At the
 conclusion of the task force, the TAG intends to adopt this document as a Finding.
 
+<aside class="note">
+
+Privacy is covered by legal frameworks and this document recognises that existing data
+protection laws take precedence for legal matters. However, because the Web is global, we
+benefit from having shared concepts to guide its evolution as a system built for the people using it
+([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
+can hopefully help all the Web's participants in different legal regimes. Our shared
+understanding is that the law is a floor, not a ceiling.
+
+</aside>
+
 </section>
 
-# Introduction {#intro}
+# Understanding Privacy Principles {#intro}
+
+Information is power. It can be used to predict and to influence people, as well as to design online
+spaces that control people's behaviour. The collection and [=processing=] of information in greater
+volume, with greater precision and reliability, with increasing interoperability across a growing
+variety of data types, and at intensifying speed is leading to an unprecedented concentration of power
+that threatens private and public liberties.
+
+What's more, automation and the increasing computerisation of all aspects of our lives
+both increase the power of information and decrease the cost of a number of intrusive
+behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
+the victim.
+
+These asymmetries of information and of automation create commanding asymmetries of power. It is not
+so much that <q>code is law</q>  but rather that code is judge, jury, and executioner.
+
+The Web is for everyone ([[?For-Everyone]]). It is <q>a platform that helps people and provides a
+net positive social benefit</q> ([[?ETHICAL-WEB]], [[?design-principles]]). One of the ways in which the
+Web serves people is by protecting them in the face of asymmetries of power and this includes
+establishing and enforcing rules to govern the power of data.
+
+The Web is a social and technical system made up of [=information flows=]. Because this document
+is specifically about [=privacy=] as it applies to the Web, it also focuses on privacy around
+information flows.
+
+<dfn data-lt="governance">Data governance</dfn> is the system of [=rules=] that regulate [=information flows=]. When
```

Comment:
No — the rules that apply at one level of an institutional situation are the governance of that space. Those rules _also_ inherit (or embed themselves in) the rules for deciding how the rules to change the rules work. And this goes levels, until you hit physics.

To give an example, let's say you and I enjoy playing a weird variant n-dimensional tic-tac-toe (nDT3). The rules of nDT3 _govern_ the game. However, we may also have agreement between ourselves as to how we decide to change those rules. A typical way would be that we would belong to an nDT3 federation which governs the rules that govern the game — that's the governance of nDT3. Then there's the governance of the nDT3 federation itself, its embedding in the governance of free associations, etc etc.

One thing that we might find hard to thread is that I very much believe that for the Web to work we're going to have to operate from a much crisper understanding of governance, how it works, how it can be legitimate, what it even means to be legitimate in a transnational context, etc. We need better than the "big companies and people burning out" method that has dominated open standards and open source. But I also don't want to include a treatise on infrastructure, commons management, and institutional analysis — even though that's the kind of intellectual rigour we probably should demand (and that I'm clumsily trying to apply) given the scale and impact of the Web.